code/modules/projectiles/guns/ballistic.dm

```diff
@@ -351,7 +351,7 @@
 			user.dropItemToGround(src, TRUE)
 			return
 		flip_cooldown = (world.time + 30)
-		user.visible_message("<span class='notice'>[user] spins the [src] around their finger by the trigger. That’s pretty badass.</span>")
+		user.visible_message("<span class='notice'>[user] spins /the [src] around their finger by the trigger. That’s pretty badass.</span>")
```

Comment:
so close, but also the is added automatically anyway

```suggestion
		user.visible_message("<span class='notice'>[user] spins [src] around their finger by the trigger. That’s pretty badass.</span>")
```

Reply:
i always forget if it defaults to a or the reeeeee